tsl/test/expected/vector_agg_param.out

```diff
@@ -21,23 +21,38 @@ select count(compress_chunk(x)) from show_chunks('pvagg') x;
 (1 row)
 
 analyze pvagg;
-explain (costs off)
+-- Uncomment to generate reference
+--set timescaledb.enable_vectorized_aggregation to off;
```

Comment:
Should this be here?

Reply:
Yes, this is just for conveniently generating the reference using the standard Postgres plans whenever you want to change this test.